scripts/fast-reboot-dump.py

```diff
@@ -158,29 +190,37 @@ def get_fdb(db, vlan_name, vlan_id, bridge_id_2_iface):
     return fdb_entries, available_macs, map_mac_ip
 
 def generate_fdb_entries(filename):
-    fdb_entries = []
+    asic_db = swsssdk.SonicV2Connector(host='127.0.0.1')
```

Comment:
>swsssdk [](start = 14, length = 7)

swsssdk is deprecated and the `import` change is alreay merged.
So I think code is broken here.

Reply:
how can we avoid this in the future? for example, can we remove swsssdk as a build or test dependency so that the unit test will automatically fail?

Reply:
The test case does not cover the problematic code, so the exception is not triggered.